cow_amm_notebook.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
```

Comment:
I would commit a version of the notebook without any of the output.
Otherwise the file gets large and diff get meaningless in git.